src/media/js/settings_app.js

```diff
@@ -19,6 +19,9 @@ define('settings_app',
         return window.location.search.indexOf('cache=false') === -1;
     }
 
+    var is_packaged_app = window.location.protocol === 'app:';
```

Comment:
not sure if settings is the right spot for this since the app doesn't really have a choice in configuring this.